.eslintrc.js

```diff
@@ -6,5 +6,8 @@ module.exports = {
     'prettier',
     'plugin:markdown/recommended',
     'plugin:json/recommended'
-  ]
+  ],
+  rules: {
+    '@typescript-eslint/no-unused-vars': 'error'
```

Comment:
shouldn't that be in any of the recommended plugins?

Reply:
I thought the same, but the remix config disables it [here](https://github.com/remix-run/remix/blob/main/packages/remix-eslint-config/rules/typescript.js#L45). I think its better to enable that rule.